src/mediapipe_internal/mediapipegraphdefinition.cpp

```diff
@@ -41,12 +42,18 @@
 namespace ovms {
 MediapipeGraphConfig MediapipeGraphDefinition::MGC;
 
+const std::string MediapipeGraphDefinition::SCHEDULER_CLASS_NAME{"Mediapipe"};
+
 const tensor_map_t MediapipeGraphDefinition::getInputsInfo() const {
-    return this->inputsInfo;
+    std::shared_lock lock(metadataMtx);
```

Comment:
```suggestion
    std::shared_lock lock(metadataMtx);
    retur this->inputsInfo;
```
would also copy the map